ert_shared/dark_storage/endpoints/ensembles.py

```diff
@@ -19,7 +19,17 @@ def post_ensemble(
 def get_ensemble(
     *, res: LibresFacade = Depends(get_res), ensemble_id: UUID
 ) -> js.EnsembleOut:
-    raise NotImplementedError
+    return js.EnsembleOut(
+        id=ensemble_id,
+        children=[],
+        parent=None,
+        experiment_id=get_id("experiment", "default"),
+        userdata={"name": get_name("ensemble", ensemble_id)},
+        size=42,
```

Comment:
why 42? Is this a magic number? If it should stay - maybe extract as variable with a proper naming?